docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
     environment:
     # See the client readme for more info on environment variables: 
     # https://github.com/usds/justice40-tool/blob/main/client/README.md
-      DATA_SOURCE: local
+      DATA_SOURCE: cdn

```

Comment:
Why is `cdn` set as the data source here?

Reply:
We want whoever checks out the codebase to point to the local tiles, which is now generating more quickly.

Reply:
The frontend Dockerfile was designed to be able to have the tiles supplied either locally (by running a full etl run) or by getting it from the CDN. This allows the developer to set this.

https://github.com/usds/justice40-tool/blob/main/client/README.md#environment-variables

Has this changed since we determined this requirement? (seems like ages ago we were talking about this when we were first trying to get docker going)

Reply:
Before we launched I was providing support to some of the folks from EPIC. They requested that it be set to the CDN to allow for the fastest way to get the app running. They believed that most people wont need the pipeline running locally. Maybe things have changed? We can discuss this more during discussion hour tmrw.

What are the advantages of running locally that could overcome the cost a longer build time?

Reply:
I agree there's value of a faster startup time with the CDN option. However there are two issues with this approach:

1. It doesn't make sense then that the Data-Pipeline docker instance is even built
2. For data folks, they might want to see how score is locally built, where the tiles are stored and modify the algorithm.

I don't think we should change the whole docker flow on this PR, let's create a new ticket where we enable the ability for a one line `docker run usds/justice40-map` and add this to the README, where users that just want to see the map without building the score, can do so.

Reply:
@jorge - agreed and thanks for explaining!